pkg/controller/bootstrap/tokencleaner.go

```diff
@@ -154,9 +155,10 @@ func (tc *TokenCleaner) processNextWorkItem(ctx context.Context) bool {
 }
 
 func (tc *TokenCleaner) syncFunc(ctx context.Context, key string) error {
+	logger := klog.FromContext(ctx)
```

Comment:
noobie question: why do we not create the logger per component and obtain it from `TokenCleaner` struct? Is it because there should be only a single place which should the logging honor ( `ctx` variable in this case) as outlined in the KEP to prevent ambiguity which one to use?

Reply:
Yes, that's exactly the problem.

Reply:
ok, thanks!